pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -67,102 +87,105 @@ def __init__(self, remote):
         """
         self.remote = remote
 
-    async def __call__(self, in_q, out_q):
+    @staticmethod
```

Comment:
Should this style be copied to other plugins?  What is the rationale here?

Reply:
Anytime you have a function that doesn't refer to self it should probably be a static method. It's a general good practice I think.